batch-submitter/Dockerfile

```diff
@@ -2,12 +2,12 @@ FROM golang:1.18.0-alpine3.15 as builder
 
 RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
 
-COPY ./l2geth /l2geth
-COPY ./go/bss-core /go/bss-core
-COPY ./go/batch-submitter/go.mod ./go/batch-submitter/go.sum /go/batch-submitter/
+COPY ./batch-submitter /go/batch-submitter
```

Comment:
Why do we copy into a `go` directory?

Reply:
It's just a nice container for all the go code - can be whatever we want.